<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadPage</class>
 <widget class="QDialog" name="UploadPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Upload</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="uploadPageLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>260</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: blue; color: white;</string>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFileButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background-color: #4CAF50; border: none; border-radius: 5px; font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="fileScrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none; background-color: transparent;</string>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #666; font-size: 10px;</string>
   </property>
   <property name="text">
    <string>Select any type of file from your computer to upload</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="encryptButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #666666; background-color: #e0e0e0; border: none; border-radius: 5px; font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Encrypt File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #666666; background-color: #e0e0e0; border: none; border-radius: 5px; font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #D32F2F; color: white; border: 2px solid #B71C1C; border-radius: 5px; padding: 5px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
